### ISSUE TITLE
/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility

### ISSUE THREAD

**Issue by liveasnotes** [mdn/sprints#2307]
# Request type
- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details
The [inserted image](https://mdn.mozillademos.org/files/13368/tabbed-info-box.png) of [Handling common **accessibility** problems](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility) not match WCAG AAA.
// contrast checker is [this](https://webaim.org/resources/contrastchecker/)

The image use [#ffffff and #a60000] or [#000000 and #ff0000]. A set of [#ffffff and #a60000] (white and red) passes WCAG AAA but [#000000 and #ff0000] (**black and red**) fails.

So, I suggest replacing original image with [this image](https://mdn.mozillademos.org/files/16918/20191022144107.png).
[Japanese version](https://developer.mozilla.org/ja/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility) already have been done it.

**Comment by liveasnotes** [comment]
in addition, we need a design guideline and it should say that use blue as main color because the color is good for accesibility and also suit MDN's color.

**Comment by chrisdavidmills** [comment]
I've fixed the image - thank you for reporting!

I don't have the time to write example guidelines and try to enforce their usage across MDN right now; I think the idea is good, but it wouldn't work in practice.
